crates/matrix-sdk-ui/src/room_list_service/room.rs

```diff
@@ -137,13 +137,27 @@ impl Room {
             .clone()
     }
 
-    /// Get the latest event in the timeline, if we already have it cached. Does
-    /// not fetch any events or calculate anything - if it's not already
-    /// available, we return None.
+    /// Get the latest event in the timeline.
+    ///
+    /// The latest event comes first from the `Timeline` if it exists.
+    /// Otherwise, it comes from the cache. This method does not fetch any
+    /// events or calculate anything — if it's not already available, we
+    /// return `None`.
     ///
     /// It's different from `Self::timeline().latest_event()` as it won't track
     /// the read marker and receipts.
```

Comment:
This part of the comment makes me wonder if it's the right thing to do. Does it mean we may return a read marker/receipt event now as the latest_event (which don't make lot of sense to render in the room list UI)?

Reply:
`latest_event` returns an event, not an item. It's defined by `TimelineItemKind`, with 2 variants: `Event` or `Virtual`. The `latest_event` getter returns an event, so it's not virtual, so it cannot be a read marker. However, I agree that the comment is misleading. Removing it.